src/main/java/com/havana/games/triovision/model/Card.java

```diff
@@ -22,14 +25,15 @@ public Card(Pawn[][] cardInfo) {
 
 	private void validate() {
 		
+		final int EXPECTED_NUMBER_OF_EMPTY_TILES = 3;
```

Comment:
This isn't a class level constant, so we should name it with regular camelCase. Also, I'm trying to be consistent, by always abbreviating "number" to "no", e.g. `getNoOfNulls`, `noOfEmptyTiles`, etc. Can you rename this to `expectedNoOfEmptyTiles` please. 

Nice touch with the `final` 😁